changelogs/fragments/653-initialize-disk-online-bugfix.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - win_initialize_disk - disks that are formatted but offline cannot be brought online without erasing them (https://github.com/ansible-collections/community.windows/issues/149).
```

Looks like the line is too long for `lint` so this will have to be

```suggestion
  - >
    win_initialize_disk - disks that are formatted but offline cannot be brought online without erasing them
    (https://github.com/ansible-collections/community.windows/issues/149).
```